src/Microsoft.TemplateEngine.Edge/EnumerableExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Edge
+{
+    internal static class EnumerableExtensions
```

Comment:
those may go to `Utils`.

Reply:
I was contemplating it, but wasn't sure what's our bar for public functions. I'll move and expose